hles/transformation/src/main/scala/org/broadinstitute/monster/dap/EnvironmentTransformations.scala

```diff
@@ -20,26 +21,48 @@ object EnvironmentTransformations {
       }
 
     // set address month
-    val addMonth = redcapEventName(0).filterNot(_.isDigit)
+    val addMonth: Option[String] = redcapEventName(0).filterNot(_.isDigit) match {
+      case "jan"  => Some("1")
+      case "feb"  => Some("2")
+      case "mar"  => Some("3")
+      case "apr"  => Some("4")
+      case "may"  => Some("5")
+      case "jun"  => Some("6")
+      case "july" => Some("7")
+      case "aug"  => Some("8")
+      case "sept" => Some("9")
+      case "oct"  => Some("10")
+      case "nov"  => Some("11")
+      case "dec"  => Some("12")
```

Comment:
i'm very worried by having discovered exactly two of these that are inexplicably four characters long instead of three. is there a place where i can see a list of all arms, for the sake of thoroughness?

Reply:
"baseline_arm_1", "dec2019_arm_1", "dec2019_secondary_arm_1", "jan2020_arm_1", "jan2020_secondary_arm_1", "feb2020_arm_1", "feb2020_secondary_arm_1", "mar2020_arm_1", "mar2020_secondary_arm_1", "apr2020_arm_1", "apr2020_secondary_arm_1", "may2020_arm_1", "may2020_secondary_arm_1", "june2020_arm_1", "june2020_secondary_arm_1", "july2020_arm_1", "july2020_secondary_arm_1", "aug2020_arm_1", "aug2020_secondary_arm_1", "sept2020_arm_1", "sept2020_secondary_arm_1", "oct2020_arm_1", "oct2020_secondary_arm_1", "nov2020_arm_1", "nov2020_secondary_arm_1", "dec2020_arm_1", "dec2020_secondary_arm_1"

Reply:
We may able to do this via the RedCap environmental project console, I believe @quazi-broad has access.

Reply:
@ThePletch alternatively there is an exportArms API call we can curl to get this data, we should be able to use that.

Reply:
> `june2020_arm_1`
🙃🙃🙃🙃🙃🙃🙃

Reply:
I know python has a library that you can feed in a string such as the monthYear above and it will return a datetime. I wonder if scala has something similar.

Reply:
we looked into using a datetime parsing library, but it ended up being harder to parse than just using a mapping like this.